crates/bench/benches/subscription.rs

```diff
@@ -45,7 +40,8 @@ fn insert_op(table_id: TableId, table_name: &str, row: ProductValue) -> Database
     DatabaseTableUpdate {
         table_id,
         table_name: table_name.to_string(),
-        ops: vec![TableOp::insert(row)],
+        inserts: vec![row],
```

Comment:
Here and elsewhere `TableOp` is replaced, is it correct that the order of operations doesn't matter? I will find it surprising that `delete + insert` == `insert + delete` because the potential of side-effects (aka triggers)

Reply:
We don't have triggers, and AFAIK have no plans to implement them in the near-to-medium-term future. If they're something we're interested in, we'll need to do some design work. In the meantime, I see no reason to make our performance worse in service of a feature that we don't have but might someday add.

Also, notably, we already shuffle the table ops relative to the order they're performed in a TX. `TxState` holds unordered sets of inserts and deletes, and has no way to reconstruct an ordered sequence of mutations.

Reply:
But we have reducers that are alike.